src/chttpd/src/chttpd_db.erl

```diff
@@ -250,20 +251,83 @@ handle_view_cleanup_req(Req, Db) ->
     ok = fabric:cleanup_index_files_all_nodes(Db),
     send_json(Req, 202, {[{ok, true}]}).
 
+
+handle_partition_design_req(#httpd{
+        path_parts=[DbName, <<"_partition">>, PartitionKey, _Design, Name, <<"_",_/binary>> = Action | _Rest]
```

Comment:
In my work I'm calling this simply "Partition". For example, you might put documents in a partition called foo. It seems redundant to call that a partition key.